pb/struct.go

```diff
@@ -0,0 +1,163 @@
+package pb
+
+import (
+	"fmt"
+	"reflect"
+
+	types "github.com/gogo/protobuf/types"
+)
+
+// ToStruct converts a map[string]interface{} to a types.Struct
+func ToStruct(v map[string]interface{}) *types.Struct {
```

Comment:
I did not use gogo protobuf very much, so could you please share a bit, why in order to create new Struct a https://godoc.org/github.com/gogo/protobuf/types#NewPopulatedStruct would not work for us?

Reply:
Because it accepts `randyStruct` but we need `map[string]interface{}`.
We parse unknown structure from yaml, the only type we can use is `map[string]interface{}`.